### DISCUSSION TITLE
Компактная панель фильтра расширений

### DISCUSSION THREAD

**Discussion by maxim-pishnyak** [JPathRu/localisation#77] [Ideas]
Для экранов 1024х768 панель фильтра (к примеру, расширений) изначально может быть однострочной (как и в английской версии).
[j4](https://user-images.githubusercontent.com/11784484/133969490-c99cac53-df39-43c5-a3e6-1d6a6f24ecf5.jpg)
Однако такое изменение затронет и другие фильтры админки. Список констант теперь уже без скобок и замыкающая список константа более лаконичной кнопки фильтра прилагается:
COM_INSTALLER_HEADING_FOLDER_ASC="Каталог по возрастанию"
COM_INSTALLER_HEADING_FOLDER_DESC="Каталог по убыванию"
COM_INSTALLER_HEADING_LANGUAGE_TAG_ASC="Тег языка по возрастанию"
COM_INSTALLER_HEADING_LANGUAGE_TAG_DESC="Тег языка по убыванию"
COM_INSTALLER_HEADING_LOCATION_ASC="Область по возрастанию"
COM_INSTALLER_HEADING_LOCATION_DESC="Область по убыванию"
COM_INSTALLER_HEADING_PACKAGE_ID_ASC="ID пакета по возрастанию"
COM_INSTALLER_HEADING_PACKAGE_ID_DESC="ID пакета по убыванию"
COM_INSTALLER_HEADING_TYPE_ASC="Тип по возрастанию"
COM_INSTALLER_HEADING_TYPE_DESC="Тип по убыванию"
COM_INSTALLER_HEADING_UPDATESITE_NAME_ASC="Сервер по возрастанию"
COM_INSTALLER_HEADING_UPDATESITE_NAME_DESC="Сервер по убыванию"
JGLOBAL_NAME_ASC="Название по возрастанию"
JGLOBAL_NAME_DESC="Название по убыванию"
JASSOCIATIONS_ASC="Связи по возрастанию"
JASSOCIATIONS_DESC="Связи по убыванию"
JAUTHOR_ASC="Автор по возрастанию"
JAUTHOR_DESC="Автор по убыванию"
JCATEGORY_ASC="Категория по возрастанию"
JCATEGORY_DESC="Категория по убыванию"
JDATE_ASC="Дата по возрастанию"
JDATE_DESC="Дата по убыванию"
JFEATURED_ASC="Избранный по возрастанию"
JFEATURED_DESC="Избранный по убыванию"
JSTAGE_ASC="Этап по возрастанию"
JSTAGE_DESC="Этап по убыванию"
JSTATUS_ASC="Состояние по возрастанию"
JSTATUS_DESC="Состояние по убыванию"
JGLOBAL_HITS_ASC="Количество просмотров по возрастанию"
JGLOBAL_HITS_DESC="Количество просмотров по убыванию"
JGLOBAL_NAME_ASC="Заголовок по возрастанию"
JGLOBAL_NAME_DESC="Заголовок по убыванию"
JGLOBAL_RATINGS_ASC="Рейтинг по возрастанию"
JGLOBAL_RATINGS_DESC="Рейтинг по убыванию"
JGLOBAL_TITLE_ASC="Заголовок по возрастанию"
JGLOBAL_TITLE_DESC="Заголовок по убыванию"
JGLOBAL_VOTES_ASC="Количество оценок по возрастанию"
JGLOBAL_VOTES_DESC="Количество оценок по убыванию"
JGRID_HEADING_ACCESS_ASC="Доступ по возрастанию"
JGRID_HEADING_ACCESS_DESC="Доступ по убыванию"
JGRID_HEADING_CAPTION_ASC="%s по возрастанию"
JGRID_HEADING_CAPTION_DESC="%s по убыванию"
JGRID_HEADING_ID_ASC="ID по возрастанию"
JGRID_HEADING_ID_DESC="ID по убыванию"
JGRID_HEADING_LANGUAGE_ASC="Язык по возрастанию"
JGRID_HEADING_LANGUAGE_DESC="Язык по убыванию"
JGRID_HEADING_ORDERING_ASC="Порядок по возрастанию"
JGRID_HEADING_ORDERING_DESC="Порядок по убыванию"
JFILTER_OPTIONS="Фильтр"

**Comment by b2z** [comment]
Хм, а кто у нас сейчас сидит на экранах 1024х768? Меньше 5 процентов пользователей.
https://www.w3schools.com/browsers/browsers_display.asp
https://gs.statcounter.com/screen-resolution-stats/desktop/worldwide

**Comment by Kostelano** [comment]
В Joomla 3 есть такая проблема, но лично я решил это путем изменения ширины списка за счет переопределения стилей шаблона Isis. Поскольку даже предложенный вариант не решил бы проблему полностью, а только 20% (инглиш более компактный язык, к сожалению).

В Joomla 4 вообще нет этой проблемы, даже с учетом использования такого маленького экрана.

К тому же, как по мне, без наличия скобок все "сливается". Поэтому я все же предпочел бы не проводить такое изменение.

**Comment by maxim-pishnyak** [comment]
Да, согласен, все сказанное резонно. Однако... пресловутый инклюзивный подход мог бы решение развернуть в сторону тех штучных процентов. А вдруг они тоже важны, как вторят тренды?
А у планшетов расширение не 1024х768? Что же? Минусовать планшетное администрирование?
Учитывая дискуссионность локализации как явления, привычным выходом остается как всегда таскание с собой с проекта на проект своего индвидуального файла ru-RU.override.ini, который во всем устраивает конкретно взятого разработчика.
